SConstruct

```diff
@@ -294,8 +297,16 @@ def seed_cluster(cp, best_plus_i, seed_id):
 def seed_cluster_size(cp, best_plus_i, seed_id):
     return len(seed_cluster(cp, best_plus_i, seed_id))
 
-def partition_metadata(part, cp, best_plus_i, seed=None, other_id=None):
+def get_alt_naive_probabilities(annotation):
+    alternatives = annotation.get('alternative-annotations', {})
+    naive_probabilities = alternatives.get('naive-seqs') if alternatives else None
+    return naive_probabilities if naive_probabilities and len(naive_probabilities) > 0 else None
+
+def partition_metadata(part, annotation_list, cp, best_plus_i, seed=None, other_id=None):
     i = cp.i_best + best_plus_i
+    seed_cluster_annotation = None
+    if i == cp.i_best and seed:
```

Comment:
Suggest `best_plus_i == 0`, since that's really what you're getting at here. Also a comment about why this check is necessary would be nice.

Reply:
@metasoarous What we are getting at is that it is the best partition aka cp.i_best which I thought was more explicitly conveyed by `i == cp.i_best` than `best_plus_i == 0`. Open to an argument for the latter though. Agreed that a comment would clarify.

Reply:
Either way. Your call.